src/utils/barcode.ts

```diff
@@ -0,0 +1,41 @@
+import { createCanvas } from "canvas";
+import * as t from "io-ts";
+import * as JsBarcode from "jsbarcode";
+import { DOMImplementation, XMLSerializer } from "xmldom";
+
+export const BarcodeOutput = t.interface({
```

Comment:
don't create a io-ts type if you don't need decode, just use a plain typescript interface